### PR TITLE
removing restriction for xero app

### PR DESCRIPTION
**Pull request by Ashutosh619-sudo**

### CODE REVIEW

File: fyle_accounting_mappings/views.py [hunk 1]
```diff
@@ -279,7 +278,7 @@ def get_queryset(self):
         }
 
         if ((source_type == 'PROJECT' and destination_type == 'CUSTOMER') or\
-            (source_type == 'CATEGORY')) and app_name != 'XERO':
```
Comment (ashwin1111) [inline fyle_accounting_mappings/views.py:282]: remove the same from stat endpoint as well

File: fyle_accounting_mappings/views.py [hunk 2]
```diff
@@ -279,7 +278,7 @@ def get_queryset(self):
...
+            (source_type == 'CATEGORY')):
             filters['active'] = True
```
Comment (ashwin1111) [inline fyle_accounting_mappings/views.py:282]: bump library version as well, increment the patch version

Comment (Ashutosh619-sudo) [reply fyle_accounting_mappings/views.py:282]: in the setup.py to 1.26.2 right?

Comment (ashwin1111) [reply fyle_accounting_mappings/views.py:282]: yes

File: fyle_accounting_mappings/views.py [hunk 3]
```diff
@@ -196,7 +196,7 @@ def get(self, request, *args, **kwargs):
         }
 
         if ((source_type == 'PROJECT' and destination_type == 'CUSTOMER') or\
-            (source_type == 'CATEGORY')) and app_name != 'XERO':
```
Comment (ashwin1111) [inline fyle_accounting_mappings/views.py:199]: remove line 188 if it's not used anywhere

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**